### PR TITLE
Docker Support

### PR DESCRIPTION
**Pull request by xbdmHQ**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
 {
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.2",
-    "bootstrap": "^4.6.1",
+    "@fortawesome/fontawesome-free": "^6.4.2",
+    "bootstrap": "^5.3.1",
```
Comment (SergiX44) [inline package.json:4]: **🔨 issue:** ‏sorry is not possible to update all the frontend frameworks without updating also the templates  and adapt the layouts. everything is designed around bs4, so the theme switcher and other components will break without proper adaptiation.

File: README.md [hunk 2]
```diff
@@ -41,13 +41,3 @@ All the installations, configuration, and usage instructions are available in th
...
-
-
-## Security Vulnerabilities
-
-If you discover a security vulnerability within XBackBone, please send an e-mail to Sergio at sergio@brighenti.me. All security vulnerabilities will be promptly addressed.
-
-## License
-This software is licensed under the <a href="https://choosealicense.com/licenses/agpl-3.0/">GNU Affero General Public License v3.0</a>, available in this repository.
-As a "copyright notice" it is sufficient to keep the small footer at the bottom of the page, also to help other people to learn about this project!
-
```
Comment (SergiX44) [inline README.md:53]: **🔨 issue:** ‏Why thi got removed?

**Review by SergiX44 — CHANGES_REQUESTED** [review]
Other the random dependency upgrade that is just wrong without updating the design, I don't get the need to use the built-in docker compose (also, using the linuxserver image instead of pointing to the local docker compose you created), thats will add just more stuff to maintain, without improving the user experience in my opinion.

**Comment by xbdmHQ** [comment]
> Other the random dependency upgrade that is just wrong without updating the design, I don't get the need to use the built-in docker compose (also, using the linuxserver image instead of pointing to the local docker compose you created), thats will add just more stuff to maintain, without improving the user experience in my opinion.
> 
> 

People is already using it, Docker is a containerized and secure convenience environment so this will help a lot of new self hosters.. the production is used for production Incase no one don't want to modified the source code, the development is for people who want to change the source code with docker-compose-development.yml

File: README.md (hunk 2 above)
Comment (xbdmHQ) [reply README.md:53]: I explain it in the commit already, but I will do it again it you mentioned the LICENSE and SECURITY in the README.me which you need to do that it's already a file and people will see it if they scroll through the source code

**Comment by SergiX44** [comment]
> People is already using it, Docker is a containerized and secure convenience environment so this will help a lot of new self hosters.. the production is used for production Incase no one don't want to modified the source code, the development is for people who want to change the source code with docker-compose-development.yml

Yes, I know what Docker is.
But since a docker image is already maintained very well by linuxserver.io, I dont see the point to create a new one here.
The development one is basically useless, since is pulling a zip from the remote repository releases.
The LICENSE does not contain any markdown, so I dont know why you renamed it.
We cant enable dependant bot without having a proper test suite to make sure that an upgrade is not causing a regression.
The .env vars contains variables that should be constants, since you commented with Dont touch.

I'm sorry, but right now I'm not going to merge this, there are too many changes that should be addressed in different PRs, and some of them would require a discussion if is worth the additional maintenance work.

**Pull request closed without merging**